OpenSim/Common/Test/testDataTable.cpp

```diff
@@ -251,6 +268,29 @@ int main() {
             ASSERT(row2[8] == 1);
         }
 
+        {
+        std::cout << "Test feeding rows of one table to another [Vec3]."
+                  << std::endl;
+        DataTableVec3 tableVec3Copy{};
+        tableVec3Copy.setColumnLabels(tableVec3.getColumnLabels());
+        for(unsigned row = 0; row < tableVec3.getNumRows(); ++row)
+            tableVec3Copy.appendRow(tableVec3.getIndependentColumn()[row],
+                                    tableVec3.getRowAtIndex(row));
+
+        ASSERT(tableVec3Copy.getNumColumns() == tableVec3.getNumColumns());
+        ASSERT(tableVec3Copy.getNumRows()    == tableVec3.getNumRows());
+        for(unsigned r = 0; r < tableVec3.getNumRows(); ++r) {
+            for(unsigned c = 0; c < tableVec3.getNumColumns(); ++c) {
+                ASSERT(tableVec3Copy.getRowAtIndex(r).getElt(0, c)[0] ==
+                           tableVec3.getRowAtIndex(r).getElt(0, c)[0]);
+                ASSERT(tableVec3Copy.getRowAtIndex(r).getElt(0, c)[1] ==
+                           tableVec3.getRowAtIndex(r).getElt(0, c)[1]);
+                ASSERT(tableVec3Copy.getRowAtIndex(r).getElt(0, c)[2] ==
+                           tableVec3.getRowAtIndex(r).getElt(0, c)[2]);
```

Comment:
Would be easier to read if we used  `ASSERT_EQUAL<SimTK::Vec3>(...)`. Also why access a row element using `.getElt(0,c)` then using `[c]` operator?